keel-slack/src/main/kotlin/com/netflix/spinnaker/keel/slack/callbacks/ManualJudgmentCallbackHandler.kt

```diff
@@ -104,8 +90,13 @@ class ManualJudgmentCallbackHandler(
         }
       }
 
+      val originalBlocks = response.message.blocks
+      //remove the first two blocks because we're replacing them
+      originalBlocks.removeFirstOrNull()
+      originalBlocks.removeFirstOrNull()
+      return newBlocks + originalBlocks
```

Comment:
Perfect! Something like that is what I had in mind!

Reply:
I was so stuck on working inside the `withBlocks` thing that I forgot it returned a list of blocks. 🤞